+bids/+util/jsonencode.m

```diff
@@ -59,8 +76,21 @@
       fprintf(fid, '%s', txt);
```

Comment:
Will it write to file with new lines and indend?

Reply:
I just checked and if you rely on Matlab built-in function there will not be any JSON "formating".

I might be wrong but I don't think those options are supported by Matlab's functions. :sob:

Reply:
My little finger told me to tell you to wait a few weeks for `PrettyPrint`.

Reply:
https://www.mathworks.com/help/matlab/ref/jsonencode.html#bvb7dgc-2